src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -38,78 +39,42 @@
 	private static final Pattern intPattern = Pattern.compile("([\\+-]?\\d+)([eE][\\+-]?\\d+)?");
 //	private static final Pattern floatPattern = Pattern.compile("([\\+-]?\\d(\\.\\d*)?|\\.\\d+)([eE][\\+-]?(\\d(\\.\\d*)?|\\.\\d+))?");
 
-	public static final int BITRATE_DEFAULT = 8000;
-	public static final int NUMBEROFCHANNELS_DEFAULT = 2;
-	public static final int BITSPERSAMPLE_DEFAULT = 16;
-	public static final int AUDIODELAY_DEFAULT = 0;
-	public static final int SAMPLEFREQUENCY_DEFAULT = 48000;
-
-	private int bitRate;
 	private int numberOfChannels;
-	private int bitsperSample;
 	private int audioDelay;
 	private int sampleFrequency;
 
 	public int getAttribute(AudioAttribute attribute) {
 		switch (attribute) {
-			case BITRATE:
```

Comment:
I seem that you lied to me when you was saying that you was wishing delete this class ?
Perhaos you did in an other commit, or still plan to do, i really don't know.